presto-parquet/src/main/java/com/facebook/presto/parquet/writer/ParquetWriter.java

```diff
@@ -121,7 +121,7 @@ public ParquetWriter(OutputStream outputStream,
 
     public long getWrittenBytes()
     {
-        return outputStream.longSize();
+        return outputStream.longSize() + getBufferedBytes();
```

Comment:
The change should happen in ParquetFileWriter, not here. If you add the buffered bytes here, the getWrittenBytes() would return the sum of written bytes and buffered bytes and won't match the function name anymore. Check how OrcFileWriter does it:
```
 @Override
    public long getWrittenBytes()
    {
        return orcWriter.getWrittenBytes() + orcWriter.getBufferedBytes();
    }
```